src/main/kotlin/io/sip3/captain/ce/capturing/PcapEngine.kt

```diff
@@ -79,6 +87,12 @@ class PcapEngine : AbstractVerticle() {
             config.getString("dev")?.let {
                 dev = it
             }
+            config.getString("dlt")?.let {
+                if (!DATA_LINK_TYPES.contains(it)) {
```

Comment:
Let's use `require()` in such cases?
`require(DATA_LINK_TYPES.contains(it)) { "Unsupported datalink type: $it" }`